src/Components/Blazor/Build/src/Microsoft.AspNetCore.Blazor.Build.csproj

```diff
@@ -1,9 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetName>Microsoft.AspNetCore.Blazor.Build.Tasks</TargetName>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
*Things to talk to SDK:* We need this for System.Text.Json. Should we do something different?

Reply:
oooh yeah we need to talk about that. It's not a safe assumption that we can use S.T.J in MSBuild until Nick tells me we can.

Reply:
I switched this to DataContractJsonSerializer. Dan helped me figure out how to use it.